appinterface/projection/crossfire/crossfire.go

```diff
@@ -344,25 +355,38 @@ func (projection *Crossfire) projectCrossfireValidatorView(
 			operatorAddress, errConverting := tmcosmosutils.ValidatorAddressFromPubAddress(projection.validatorAddressPrefix, msgVoteCreated.Voter)
 
 			if errConverting != nil {
-				return fmt.Errorf("error In converting voter address to validator address: s%v", errConverting)
+				return fmt.Errorf("error In converting voter address to validator address: %v", errConverting)
 			}
 
 			errCheckingTask := projection.checkTaskNetworkProposalVote(crossfireValidatorsView, operatorAddress, blockTime)
 
 			if errCheckingTask != nil {
-				return fmt.Errorf("error In checking Network proposal vote: s%v", errCheckingTask)
+				return fmt.Errorf("error In checking Network proposal vote: %v", errCheckingTask)
 			}
 			// Update the proposed ID against the voter in Database
 			voted_proposal_id_db_key := constants.VOTED_PROPOSAL_ID + constants.DB_KEY_SEPARATOR + msgVoteCreated.Voter
 
 			proposalIdAsInt64, errConversion := strconv.ParseInt(msgVoteCreated.ProposalId, 10, 64)
 			if errConversion != nil {
-				return fmt.Errorf("error converting ProposalID to int64: s%v", errConversion)
+				return fmt.Errorf("error converting ProposalID to int64: %v", errConversion)
 			}
 			errUpdateValidatorStats := crossfireValidatorStatsView.Set(voted_proposal_id_db_key, proposalIdAsInt64)
 
 			if errUpdateValidatorStats != nil {
-				return fmt.Errorf("error Updating ProposalID for the voter: s%v", errUpdateValidatorStats)
+				return fmt.Errorf("error Updating ProposalID for the voter: %v", errUpdateValidatorStats)
+			}
+		} else if msgTransactionCreated, ok := event.(*event_usecase.TransactionCreated); ok {
```

Comment:
Refer to https://github.com/crypto-com/chain-indexing/blob/master/appinterface/projection/transaction/transaction.go#L74, it should be
```
if transactionCreatedEvent
```
?

Reply:
Yes yes correct.